client/me/purchases/concierge-banner/index.jsx

```diff
@@ -107,6 +110,7 @@ class ConciergeBanner extends Component {
 			<>
 				<TrackComponentView eventName="calypso_purchases_concierge_banner_view" />
 				<Banner
+					className={ 'concierge-banner' }
```

Comment:
Style nit: this isn't a dynamic value, so you can just use `"concierge-banner"` as a string.
```suggestion
					className="concierge-banner"
```

Reply:
This would be a good candidate for a `prettier` rule. Currently, it just maintains what exists already, which can be with or without brackets. I've manually fixed it and a couple of others for now.